README.md

```diff
@@ -63,3 +63,34 @@ After modify the fullnode variables, overwrite the existing setting:
 ```bash
 $ ansible-playbook iri.yml -e overwrite=yes
 ```
+
+To change all service status:
+
+```bash
+# restart service
+$ ansible-playbook manage.yml -e systemd_status=restarted
+# start service
+$ ansible-playbook manage.yml -e systemd_status=started
+# stop service
+$ ansible-playbook manage.yml -e systemd_status=stopped
+```
```

Comment:
I think these can be simplified.

---

For example:

To change all service status:

```bash
$ ansible-playbook manage.yml -e systemd_status=[status]
```
The acceptable **[status]** are
- `started`
- `restarted`
- `stopped`

---

The others can be modified as well.

Reply:
Updated. 👍